pycode/memilio-epidata/setup.py

```diff
@@ -68,8 +68,8 @@ def run(self):
     long_description='',
     test_suite='memilio.epidata_test',
     install_requires=[
-        'pandas<1.2.0',
-        'matplotlib<3.4',
+        'pandas>=1.2.2',
```

Comment:
Yes, for smaller versions pandas contains a bug so that some exel files are read as if they were empty. See also issue #201. This bug usally disappears if the exel file is opened and saved again with some exel reader. I read up in which pandas version this bug was fixed and for me using pandas version>=1.2.2 solved the problem. Can you check whether you have this issue still with the newer pandas version?

Reply:
Looks good to me!